### PR TITLE
refactor(tests) - Tests to fixtures - mergeSiblingVars

### PR DESCRIPTION
**Pull request by vigneshshanmugam**

### CODE REVIEW

File: packages/babel-plugin-transform-merge-sibling-variables/__tests__/fixtures/not-concat-block-scoped/actual.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+let i = 0;
```
Comment (boopathi) [inline packages/babel-plugin-transform-merge-sibling-variables/__tests__/fixtures/not-concat-block-scoped/actual.js]: rename file - `block-scoped-for` ?

File: packages/babel-plugin-transform-merge-sibling-variables/__tests__/fixtures/not-concat-const-for-loop/actual.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+const j = 0;
+for (const x = 0; ; ) console.log(j + x);
```
Comment (boopathi) [inline packages/babel-plugin-transform-merge-sibling-variables/__tests__/fixtures/not-concat-const-for-loop/actual.js]: move this to previous test - as it tests block scoped vars merged to for loop

File: packages/babel-plugin-transform-merge-sibling-variables/__tests__/fixtures/not-lift-on-destructing/actual.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+for (var i = 0; i < 0; i++) {
```
Comment (boopathi) [inline packages/babel-plugin-transform-merge-sibling-variables/__tests__/fixtures/not-lift-on-destructing/actual.js]: filename: `destructuring-var-lift` ?

File: packages/babel-plugin-transform-merge-sibling-variables/__tests__/fixtures/not-concat-const-for-loop/actual.js (hunk 2 above)
Comment (vigneshshanmugam) [reply packages/babel-plugin-transform-merge-sibling-variables/__tests__/fixtures/not-concat-const-for-loop/actual.js]: cool

**Comment by boopathi** [comment]
Looks like formatting errors - 

```sh
$(yarn bin)/jest packages/babel-plugin-merge-sibling-variables -- --update-fixtures
```

will fix them

**Pull request merged**